framework/tests/viewpointtest/SuperConstructorCalls.java

```diff
@@ -18,6 +18,7 @@ public Inner() {
         }
 
         public Inner(@Top Object objTop) {
+            // :: error: (argument.type.incompatible)
```

Comment:
A comment would be helpful here. I assume the constructor is `@Top` by default, which then leads to `@Lost` in the super constructor signature.

Reply:
Added!